gitrevise/odb.py

```diff
@@ -43,6 +45,7 @@ def __init__(self, stderr: str) -> None:
 
 
 T = TypeVar("T")  # pylint: disable=invalid-name
+_FILE = Union[None, int, IO[Any]]
```

Comment:
can we call this `File` to be consisten with the typing module?

Reply:
For context, this definition is directly inlined from `subprocess.pyi`'s as the type of that `Popen` argument: https://github.com/python/typeshed/blob/98afaa4c76d4279846f278eb5be8b99c391e6a99/stdlib/subprocess.pyi#L24

It's inlined here because it's not an exported type name, so I wasn't certain it would be importable in all supported versions. Alternatively it could also be done as:

```
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from subprocess import _FILE
```

My only minor concern with calling it `File` is increasing drift from `Popen`'s type.

Preference between:
  * Leave it as is
  * Import it
  * Change name to `File`

?